apps/amiapp_flutter/ios/Podfile

```diff
@@ -1,7 +1,10 @@
-# Import Ruby functions from native SDK to more easily allow installing non-production SDK builds.
+# ------------- 
+# This code only used internally for Customer.io testing
 require 'open-uri'
-IO.copy_stream(URI.open('https://raw.githubusercontent.com/customerio/customerio-ios/main/scripts/cocoapods_override_sdk.rb'), "/tmp/override_cio_sdk.rb")
+IO.copy_stream(URI.open('https://raw.githubusercontent.com/customerio/customerio-ios/v2/scripts/cocoapods_override_sdk.rb'), "/tmp/override_cio_sdk.rb")
```

Comment:
do we still need this, now that we are pointing towards production?

Reply:
I think you're asking if we should remove this block of code that installs the script? 

This PR fixes the script that was already in the `Podfile`. So I didn't add anything new to this file.